src/mixins/filing-template-mixin.ts

```diff
@@ -592,6 +594,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
       this.setNewResolutionDates(cloneDeep(
         filing.correction.shareStructure?.resolutionDates || []
       ))
+      // store Provisions Removed
+      this.setProvisionsRemoved(filing.correction.provisionsRemoved || null)
```

Comment:
I agree with this since that's what the state variable is initialized to.

In retrospect, maybe we should have initialized state (and here) to False instead, but this is falsey so it's fine. (I do note that if `filing.correction.provisionsRemoved` is False, the state is set to null not False.)